### DISCUSSION TITLE
Static html export to GitHub page?

### DISCUSSION THREAD

**Discussion by francois-baptiste** [avitorio/outstatic#123] [Q&A] [answered]
Hey, thank you for maintaining outstatic!
It would be nice to have a way for generating a static html export.
This way the website could be hosted on GitHub Pages or on any other static website hoster like AWS S3.

**Comment by avitorio** [comment, marked as answer]
Bonjour :)
Thanks for the suggestion. I think this can be done using the `export` option on Next.js. 
https://nextjs.org/docs/app/building-your-application/deploying/static-exports

**Reply by francois-baptiste** [reply]
Yes indeed!
I manage to do the export manually by adding the proposed next.config.js and by removing the `/src/pages/api/outstatic` and `/src/pages/outstatic` pages!

**Reply by bchristie** [reply]
@francois-baptiste May want to check out https://github.com/avitorio/outstatic/discussions/237. At the very least offers a work-around.
